src/java/org/apache/cassandra/index/sai/disk/v1/MemtableIndexWriter.java

```diff
@@ -56,28 +55,25 @@ public class MemtableIndexWriter implements PerIndexWriter
 {
     private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    private final IndexDescriptor indexDescriptor;
-    private final IndexContext indexContext;
+    private final ComponentGroup.Writer groupWriter;
```

Comment:
Renamed those. Fwiw, I renamed `Writer` to `ForWrite` (and same for reader) because those weren't really the "writers" per-se, but rather a sort of view of the component used to open writers to it. Anyway, I'm probably thinking too hard about all this :)